src/app/services/blockchain.service.ts

```diff
@@ -44,11 +57,32 @@ export class BlockchainService {
     return this.blocks(1).map(blocks => blocks[0]);
   }
 
-  progress() {
+  private retrieveInputAddress(input: string) {
+    return this.apiService.get('uxout', {uxid: input});
+  }
+
+  private loadBlockchainBlocks() {
+    setTimeout(() => IntervalObservable
```

Comment:
I did some testing and it seems that the procedure works correctly without needing to wait 3 seconds. Is there a particular case in which this is necessary?

Reply:
It was copied from the desktop wallet. Will be removed in next commit.